### DISCUSSION TITLE
Possible to add unverfied extension?

### DISCUSSION THREAD

**Discussion by IamReallyOverrated** [filips123/PWAsForFirefox#531] [Q&A] [answered]
Is it possible to add add-ons from file in any way? I tried to in the normal way, but it tells me it's impossible due to the add-on not being verified. Any way around this?

**Comment by filips123** [comment, marked as answer]
Which addon do you want to install? If the addon has been signed, you can just install it normally, as PWAsForFirefox doesn't change the behaviour of addon installation. If the addon is not signed, you can't permamently install it, just like in normal Firefox. If this is the case, you will have to replace your runtime Firefox with ESR, Developer Edition or Nightly, which allow you to [disable signature checks](https://extensionworkshop.com/documentation/publish/signing-and-distribution-overview/).
